specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-04-01-preview/TenantActivityLogAlerts.json

```diff
@@ -0,0 +1,543 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-04-01-preview",
+    "title": "Tenant Level Activity Log Alert Rules Resource Provider",
```

Comment:
>"Tenant Level Activity Log Alert Rules Resource Provider", [](http://example.com/codeflow?start=13&length=58)

nit, Is that right, or is it all under the umbrella of the AlertsManagement resource provider? This probably mainly effects how docs end up getting organized.